components/StoryCardData.jsx

```diff
@@ -0,0 +1,14 @@
+class StoryCardData {
```

Comment:
Is this needed? we dont use class componenets so can remove this file

Reply:
yep, forgot to remove it. Pretty new to JS and thought that we have to declare classes or structs for custom variables :)